### ISSUE TITLE
Flutter app can't be build after latest update in master channel.

### ISSUE THREAD

**Issue by DavidRuland** [flutter/flutter#69882]
Hi, 
I have a Flutter app that works fine on web, android and iPhone. However after I update to Flutter version 1.24.0-8.0.pre.97 in  master channel I can't build app for android and iPhone. It works to build for android and iPhone in another project with same code, pubspec and pods without web. My project are using Firebase and google maps for web, android and iPhone. I have already done Flutter clean, rm-rf build, rm -Rf ios/Pods, rm -Rf ios/.symlink, rm -Rf Pods/ Podfile.lock, delete pubspec.lock , pod cache clean —all, Flutter get packages and Flutter build for android or iOS. But it doesn't solve my build problem for android and iPhone. Something seems wrong with the latest update in master channel and does anyone have a solution for my problem?

**Steps to reproduce**

1. Flutter build for android and iPhone
2. Flutter clean
3. rm-rf build
4. rm -Rf ios/Pods
5. rm -Rf ios/.symlink
6. rm -Rf Pods/ Podfile.lock
7. pod cache clean —all
8. Flutter get packages
9. Flutter build android and iPhone.

**Logs**

```
Command PhaseScriptExecution failed with a nonzero exit code
    gRPC-C++-exgqhndeehyfpuavmxdgzbvyzheu
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
   Could not build the precompiled application for the device.
   Error launching application on iPhone.
```
**Flutter doctor**

```
[flutter] flutter doctor -v
[✓] Flutter (Channel master, 1.24.0-8.0.pre.97, on Mac OS X 10.15.7 19H2 darwin-x64, locale sv-SE)
    • Flutter version 1.24.0-8.0.pre.97
    • Framework revision 4c24eb10b8 (4 hours ago), 2020-11-05 11:18:00 +0100
    • Engine revision e66a720137
    • Dart version 2.12.0 (build 2.12.0-21.0.dev)

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/davidruland/Library/Android/sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses

[✓] Xcode - develop for iOS and macOS (Xcode 12.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.1, Build version 12A7403
    • CocoaPods version 1.10.0

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] Connected device (3 available)
    • iPhone (mobile)  • ios • iOS 14.1
    • Web Server (web) • web-server • web-javascript • Flutter Tools
    • Chrome (web)  • chrome • web-javascript • Google Chrome 86.0.4240.183

! Doctor found issues in 1 category.
exit code 0
```
**Pubspec partial**

```
version: 1.0.0+1

environment:
  sdk: ">=2.7.0 <3.0.0"

dependencies:
  flutter:
    sdk: flutter
  sqflite: any
  path_provider: any
  universal_platform: ^0.1.3
  responsive_flutter: ^0.0.4
  flutter_screenutil: ^1.1.0
  responsive_widgets: ^2.0.0+1
  responsive_framework: ^0.0.9
  responsive_builder: ^0.3.0
  intl: ^0.16.1
  percent_indicator: "^1.0.13"
  scoped_model: ^1.0.1
  assets_audio_player: ^1.0.1
  screen: ^0.0.5
  flutter_launcher_icons: "^0.7.0"
  shared_preferences: ^0.5.3+1
  persistent_bottom_nav_bar: any
  custom_navigator: ^0.3.0
  flutter_svg: ^0.18.0
  http: ^0.12.0+1
  async: ^2.4.1
  stacked: ^1.6.0
  stacked_services: ^0.4.3
  provider: ^4.3.1
  get_it: ^4.0.2
  image_picker: ^0.6.7+2
  cached_network_image: ^2.2.0+1
  firebase_core: ^0.4.0+9
  firebase_analytics: ^5.0.2
  firebase_auth: ^0.16.1
  firebase_storage: ^3.1.6
  cloud_firestore: ^0.13.6
  firebase: ^7.3.2
  firebase_web: ^5.0.9
  google_maps: ^3.4.5
  google_maps_flutter: ^0.5.27+3
  geolocator: ^5.3.1
  flutter_polyline_points: ^0.2.4
  json_serializable: ^3.3.0
  search_map_place: ^0.3.0
  carousel_slider: ^2.2.1
  dynamic_theme: ^1.0.1
```
**index.html partial**

```
<!DOCTYPE html>
<html>
<head>
  <meta charset="UTF-8">
  <meta content="IE=Edge" http-equiv="X-UA-Compatible">
  <meta name="description" content="A new Flutter project.">

  <!-- iOS meta tags & icons -->
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  <meta name="apple-mobile-web-app-title" content="web_template">
  <link rel="apple-touch-icon" href="icons/Icon-192.png">

  <!-- Favicon -->
  <link rel="icon" type="image/png" href="favicon.png"/>

  <title>web_template</title>
  <link rel="manifest" href="manifest.json">
</head>
<body>
  <!-- This script installs service_worker.js to provide PWA functionality to
     application. For more information, see:
     https://developers.google.com/web/fundamentals/primers/service-workers -->
  <script>
    if ("serviceWorker" in navigator) {
      window.addEventListener("load", function () {
        navigator.serviceWorker.register("flutter_service_worker.js");
      });
    }
  </script>
  <!-- The core Firebase JS SDK (always required for using any firebase service) -->
  <script src="https://www.gstatic.com/firebasejs/7.23.0/firebase-app.js"></script>
  <!-- Firebase Auth SDK -->
  <script src="https://www.gstatic.com/firebasejs/7.23.0/firebase-auth.js"></script>
  <script src="https://www.gstatic.com/firebasejs/7.23.0/firebase-firestore.js"></script>
  <script src="https://www.gstatic.com/firebasejs/7.23.0/firebase-analytics.js"></script>
  <scrips src = "https://www.gstatic.com/firebasejs/7.23.0/firebase-storage.js"></scrips>
  <script>
  <script src="https://maps.googleapis.com/maps/api/js?key=your-key-here"></script>
  <script src="main.dart.js" type="application/javascript"></script>
</body>
</html>
```

**App delegate**
```
import UIKit
import Flutter
import GoogleMaps

@UIApplicationMain
@objc class AppDelegate: FlutterAppDelegate {
  override func application(
    _ application: UIApplication,
    didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
  ) -> Bool {
    GMSServices.provideAPIKey("your-key-here")
    GeneratedPluginRegistrant.register(with: self)
    return super.application(application, didFinishLaunchingWithOptions: launchOptions)
  }
}
```
**Info.plist partial**

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
    <key> io.flutter.embedded_views_preview</key>
    <true/>
    <key>NSLocationAlwaysUsageDescription</key>
    <string>Your location is required for accurate notifactions benefits for you. You can also disable it with no problem </string>
    <key>NSLocationWhenInUseUsageDescription</key>
    <string>Your location is required for accurate notifaction benefits for you. You can disable it with no problem </string>
    <key>NSLocalNetworkUsageDescription</key>
    <string>Template usage local network to connect with other devices on your network</string>
    <key>UILaunchStoryboardName</key>
    <string>LaunchScreen</string>
    <key>UIMainStoryboardFile</key>
    <string>Main</string>
    <key>UISupportedInterfaceOrientations</key>
    <array>
        <string>UIInterfaceOrientationPortrait</string>
        <string>UIInterfaceOrientationLandscapeLeft</string>
        <string>UIInterfaceOrientationLandscapeRight</string>
    </array>
    <key>UISupportedInterfaceOrientations~ipad</key>
    <array>
        <string>UIInterfaceOrientationPortrait</string>
        <string>UIInterfaceOrientationPortraitUpsideDown</string>
        <string>UIInterfaceOrientationLandscapeLeft</string>
        <string>UIInterfaceOrientationLandscapeRight</string>
    </array>
    <key>UIViewControllerBasedStatusBarAppearance</key>
    <false/>
</dict>
</plist>
```

**Comment by TahaTesser** [comment]
Hi @DavidRuland 
Can you please provide your your `flutter build ios --verbose` in a plain text file (drop it in the comment below and a complete reproducible minimal code sample
Thank you

**Comment by DavidRuland** [comment]
Yes, sorry for late answear. This message comes in latest version of beta and master channel and here comes flutter build iOS --verbose log.
Unhandled exception:
FileSystemException(uri=org-dartlang-untranslatable-uri:dart%3Ahtml; message=StandardFileSystem only supports file:* and data:* URIs)
#0      StandardFileSystem.entityForUri (package:front_end/src/api_prototype/standard_file_system.dart:33:7)
#1      asFileUri (package:vm/kernel_front_end.dart:657:37)
<asynchronous suspension>
#2      writeDepfile (package:vm/kernel_front_end.dart:825:21)
<asynchronous suspension>
#3      FrontendCompiler.compile (package:frontend_server/frontend_server.dart:583:15)
<asynchronous suspension>
#4      _FlutterFrontendCompiler.compile (package:flutter_frontend_server/server.dart:43:22)
#5      starter (package:flutter_frontend_server/server.dart:180:27)
#6      main (file:///opt/s/w/ir/cache/builder/src/flutter/flutter_frontend_server/bin/starter.dart:13:30)
#7      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:299:32)
#8      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:168:12)
Command PhaseScriptExecution failed with a nonzero exit code
Showing All Messages
../../../.pub-cache/hosted/pub.dartlang.org/google_maps-3.4.5/lib/src/google_maps_src.dart:19:8: Error: Not found: 'dart:html'

**Comment by TahaTesser** [comment]
Hi @DavidRuland 
Can you please provide complete logs and a minimal code sample as requested to better address the issue https://github.com/flutter/flutter/issues/69882#issuecomment-723001430
Thank you

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
